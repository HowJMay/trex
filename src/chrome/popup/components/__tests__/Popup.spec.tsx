import * as React from 'react';
import renderer from 'react-test-renderer';
import Popup from '../popup';

describe('Popup', () => {
  test('Should mount the Popup', () => {
    const component = renderer.create(<Popup publicKey={'pub-key'} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
