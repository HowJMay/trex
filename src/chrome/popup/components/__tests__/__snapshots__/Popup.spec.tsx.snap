// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popup Should mount the Popup 1`] = `
<div>
  Loading...
</div>
`;

exports[`Popup Should mount the Popup 2`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="MuiPaper-root MuiCard-root makeStyles-container-2 MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      className="MuiPaper-root MuiAlert-root MuiAlert-standardError MuiPaper-elevation0"
      role="alert"
    >
      <div
        className="MuiAlert-icon"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSvgIcon-fontSizeInherit"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M11 15h2v2h-2zm0-8h2v6h-2zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
          />
        </svg>
      </div>
      <div
        className="MuiAlert-message"
      >
        <div
          className="MuiTypography-root MuiAlertTitle-root MuiTypography-body1 MuiTypography-gutterBottom"
        >
          Error
        </div>
        Extension isn't initialized yet â€”
         
        <strong>
          Access
           
          <a
            href="https://www.youtube.com"
            target="_blank"
          >
            yutube.com
          </a>
          .
        </strong>
      </div>
    </div>
    <div
      className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      <div
        className="MuiCardActions-root MuiCardActions-spacing"
        style={
          Object {
            "textAlign": "center",
          }
        }
      >
        <a
          aria-disabled={false}
          className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textSecondary"
          href="/index.html"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          target="_blank"
        >
          <span
            className="MuiButton-label"
          >
            Dashboard
          </span>
          <span
            className="MuiTouchRipple-root"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;
